tests/support/benchmark.tcl

```diff
@@ -1,5 +1,19 @@
+proc redisbenchmark_tls_config {testsdir} {
```

Comment:
I think we also disabled a bunch of tests from using TLS because benchmark support didn't exist.

Reply:
@madolson do you mean somewhere in this repo? or in AWS?

Reply:
I should have probably looked before commenting, looks like they were fixed during the original PR.